yang-semver/draft-ietf-netmod-yang-semver.txt

```diff
@@ -6,26 +6,26 @@ Network Working Group                                     J. Clarke, Ed.
 Internet-Draft                                            R. Wilton, Ed.
 Updates: 8407 (if approved)                          Cisco Systems, Inc.
 Intended status: Standards Track                               R. Rahman
-Expires: 4 April 2024                                          Graphiant
+Expires: 23 August 2024                                          Equinix
                                                               B. Lengyel
                                                                 Ericsson
                                                                J. Sterne
                                                                    Nokia
                                                                B. Claise
                                                                   Huawei
-                                                          2 October 2023
+                                                        20 February 2024
 
 
                         YANG Semantic Versioning
-                    draft-ietf-netmod-yang-semver-12
+                    draft-ietf-netmod-yang-semver-13
 
 Abstract
 
-   This document specifies a scheme and guidelines for applying an
-   extended set of semantic versioning rules to revisions of YANG
-   artifacts (e.g., modules and packages).  Additionally, this document
-   defines an RFCAAAA-compliant revision-label-scheme for this YANG
-   semantic versioning scheme.
+   This document specifies a YANG extension along with guidelines
+   guidelines for applying an extended set of semantic versioning rules
+   to revisions of YANG artifacts (e.g., modules and packages).
```

Comment:
Is it ok to mention packages in the abstract?

Reply:
I think so given that there are no references here.  I do link to this informatively later.